template-marketplace/hertzbeat-template-hub/src/main/resources/application.properties

```diff
@@ -0,0 +1,47 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+server.address=localhost
+
+spring.application.name=hetzbeat-template-hub
+
+server.port=8080
+
+server.servlet.context-path=/api
+
+spring.datasource.url=jdbc:mysql://localhost:3306/hertzbeat_template_hub
+spring.datasource.username=root
+spring.datasource.password=root
+spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
+
+# 配置使用的文件存储实现
+file.storage.type=minio
+#file.storage.type=local
+
+address=http://localhost:8080
+
+# 本地存储的路径
+local.storage.path=/path/to/local/storage
+
+# MinIO 配置
+minio.endpoint=http://127.0.0.1:9000
+minio.accessKey=CMULmgc3rfbnhmOCpXci
+minio.secretKey=Mw9wnljHl4RtVBGcipcmDTArdCx75lAwaMJeLXGC
+minio.bucketName=hertzbeat-data
+
+spring.jpa.properties.hibernate.hbm2ddl.auto=update
+spring.jpa.properties.hibernate.format_sql=true
+spring.jpa.show-sql=true
+spring.jpa.open-in-view=true
```

Comment:
hi, it is recommended to use the yml format consistent with other projects

Reply:
Ok, I got it.

Reply:
> hi, it is recommended to use the yml format consistent with other projects

Ok, I got it.